src/include/sof/drivers/dmic.h

```diff
@@ -32,16 +32,12 @@
 #define DMIC_HW_CONTROLLERS	2
 #define DMIC_HW_IOCLK		24000000
 #define DMIC_HW_FIFOS		2
-#endif
-
-#if CONFIG_SUECREEK
+#elif CONFIG_SUECREEK
```

Comment:
It looks incomplete. apl should be replaced with 1.5 and cnl with 1.8 (with special settings for sue inside).